dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ResourcesController.java

```diff
@@ -625,46 +623,6 @@ public Result deleteUdfFunc(@Parameter(hidden = true) @RequestAttribute(value =
         return udfFuncService.delete(loginUser, udfFuncId);
     }
 
-    /**
-     * authorized file resource list
-     *
-     * @param loginUser login user
-     * @param userId user id
-     * @return authorized result
-     */
-    @Operation(summary = "authorizedFile", description = "AUTHORIZED_FILE_NOTES")
-    @Parameters({
-            @Parameter(name = "userId", description = "USER_ID", required = true, schema = @Schema(implementation = int.class, example = "100"))
-    })
-    @GetMapping(value = "/authed-file")
-    @ResponseStatus(HttpStatus.CREATED)
-    @ApiException(AUTHORIZED_FILE_RESOURCE_ERROR)
-    public Result authorizedFile(@Parameter(hidden = true) @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
```

Comment:
Should change the ui together?

Reply:
No, this has already been removed from ui.